src/main/java/com/app/katchup/Meeting/MeetingController.java

```diff
@@ -0,0 +1,22 @@
+package com.app.katchup.Meeting;
+
+import com.app.katchup.Meeting.model.Meeting;
+import com.app.katchup.Meeting.model.Poll;
+import com.app.katchup.MeetingResponse.model.Decision;
+import com.app.katchup.MeetingResponse.model.MeetingResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class MeetingController {
+    @Autowired
+    MeetingService meetingService;
+
+    @PostMapping("/meeting")
+    public ResponseEntity<Meeting> postMeeting(@RequestBody Meeting meeting){
+        meetingService.createMeeting(meeting);
+        return new ResponseEntity<>(meeting, HttpStatus.OK);
```

Comment:
get the returned meeting object from above line and return it along with responseentity

Reply:
By default, you are returning meeeting object though the inside method returns some meeting for the host. This should not happen. You should get the response from the create meeting and check it if it is null, If it is null, return the error to the user if not return meeting created and the created object